boards/samd20-xpro/include/periph_conf.h

```diff
@@ -0,0 +1,319 @@
+        .gclk_src = SAM0_GCLK_MAIN,
+    },
+    {    /* EXT1 */
+        .dev      = &SERCOM4->USART,
+        .rx_pin   = GPIO_PIN(PB,9),
+        .tx_pin   = GPIO_PIN(PB,8),
+#ifdef MODULE_PERIPH_UART_HW_FC
+        .rts_pin  = GPIO_UNDEF,
+        .cts_pin  = GPIO_UNDEF,
+#endif
+        .mux      = GPIO_MUX_D,
+        .rx_pad   = UART_PAD_RX_1,
+        .tx_pad   = UART_PAD_TX_0,
+        .flags    = UART_FLAG_NONE,
+        .gclk_src = SAM0_GCLK_MAIN,
+    },
+};
+
+/* interrupt function name mapping */
+#define UART_0_ISR          isr_sercom3
+#define UART_1_ISR          isr_sercom4
+
+#define UART_NUMOF          ARRAY_SIZE(uart_config)
+/** @} */
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+static const spi_conf_t spi_config[] = {
+    {   /* EXT1 */
+        .dev      = &SERCOM0->SPI,
+        .miso_pin = GPIO_PIN(PA, 4),
+        .mosi_pin = GPIO_PIN(PA, 6),
+        .clk_pin  = GPIO_PIN(PA, 7),
+        .miso_mux = GPIO_MUX_D,
+        .mosi_mux = GPIO_MUX_D,
+        .clk_mux  = GPIO_MUX_D,
+        .miso_pad = SPI_PAD_MISO_0,
+        .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
+        .gclk_src = SAM0_GCLK_MAIN,
+    },
+    {   /* EXT2 */
+        .dev      = &SERCOM1->SPI,
+        .miso_pin = GPIO_PIN(PA, 16),
+        .mosi_pin = GPIO_PIN(PA, 18),
+        .clk_pin  = GPIO_PIN(PA, 19),
+        .miso_mux = GPIO_MUX_C,
+        .mosi_mux = GPIO_MUX_C,
+        .clk_mux  = GPIO_MUX_C,
+        .miso_pad = SPI_PAD_MISO_0,
+        .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
+        .gclk_src = SAM0_GCLK_MAIN,
+    },
+    {   /* EXT3 */
+        .dev      = &SERCOM5->SPI,
+        .miso_pin = GPIO_PIN(PB, 16),
+        .mosi_pin = GPIO_PIN(PB, 22),
+        .clk_pin  = GPIO_PIN(PB, 23),
+        .miso_mux = GPIO_MUX_C,
+        .mosi_mux = GPIO_MUX_D,
+        .clk_mux  = GPIO_MUX_D,
+        .miso_pad = SPI_PAD_MISO_0,
+        .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
+        .gclk_src = SAM0_GCLK_MAIN,
+    },
+};
+
+#define SPI_NUMOF           ARRAY_SIZE(spi_config)
+/** @} */
+
+/**
+ * @name I2C configuration
+ * @{
+ */
+static const i2c_conf_t i2c_config[] = {
+    {
+        .dev      = &(SERCOM2->I2CM),
+        .speed    = I2C_SPEED_NORMAL,
+        .scl_pin  = GPIO_PIN(PA, 9),
+        .sda_pin  = GPIO_PIN(PA, 8),
+        .mux      = GPIO_MUX_D,
+        .gclk_src = SAM0_GCLK_MAIN,
+        .flags    = I2C_FLAG_NONE
+     }
+};
+#define I2C_NUMOF          ARRAY_SIZE(i2c_config)
+/** @} */
+
+/**
+ * @name RTT configuration
+ * @{
+ */
+#ifndef RTT_FREQUENCY
+#define RTT_FREQUENCY       (32768U)    /* in Hz. For changes see `rtt.c` */
+#endif
+/** @} */
+
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_0_EN            1
+
+#if PWM_0_EN
+/* PWM0 channels */
+static const pwm_conf_chan_t pwm_chan0_config[] = {
+    /* GPIO pin, MUX value, TCC channel */
+    { GPIO_PIN(PA, 14), GPIO_MUX_E, 0 },
+};
+#endif
+
+/* PWM device configuration */
+static const pwm_conf_t pwm_config[] = {
+#if PWM_0_EN
+    { .tim  = TC_CONFIG(TC3),
```

Comment:
This will conflict with `timer_dev(0)`. Don't we have a better alternative ?

Reply:
xtimer will use `TIMER_DEV(1)`, so 0 should be  usable.
I really like to have a PWM output connected to the LED if possible, so it's easy to visually verify that it's working.
PA14 only has TC3 as PWM source, but we might use a different timer for `TIMER_DEV(0)`.

Reply:
7e7eb1b147795317e0a0c54ac8a4c06955a739d8 frees up TC2/TC3.
Unfortunately I don't have the board with me right now.

Reply:
@moguilevski confirmed that `examples/timer_periodic_wakeup` still works with the new timer configuration

```
2020-12-18 17:53:16,487 # Connect to serial port /dev/ttyACM0
Welcome to pyterm!
Type '/exit' to exit.
2020-12-18 17:53:17,493 # slept until 1005816
2020-12-18 17:53:18,314 # slept until 2005816
2020-12-18 17:53:19,322 # slept until 3005816
2020-12-18 17:53:20,330 # slept until 4005816
2020-12-18 17:53:21,338 # slept until 5005816
2020-12-18 17:53:22,345 # slept until 6005816
2020-12-18 17:53:23,354 # slept until 7005816
2020-12-18 17:53:24,362 # slept until 8005816
2020-12-18 17:53:25,370 # slept until 9005816
2020-12-18 17:53:26,377 # slept until 10005816
```